cookbook/docs/getting_started/tasks_and_workflows.md

```diff
@@ -0,0 +1,238 @@
+---
+jupytext:
+  formats: md:myst
+  text_representation:
+    extension: .md
+    format_name: myst
+kernelspec:
+  display_name: Python 3
+  language: python
+  name: python3
+---
+
+(getting_started_tasks_and_workflows)=
+
+# Tasks, Workflows and LaunchPlans
+
+In the {ref}`Introduction to Flyte <getting_started_index>`, we got a basic sense
+of how Flyte works by creating a few tasks and a simple model-training workflow.
+In this guide, you'll learn more about how tasks and workflows fit into the Flyte
+programming model.
+
+## Tasks
+
+Flyte tasks are the core building blocks of larger, more complex workflows.
+
+### Tasks are Containerized Blocks of Compute
+
+You can think of Flyte tasks as containerized blocks of compute. When tasks are
+run in a Flyte backend, they are actually isolated from all other tasks.
+Consider this simple task:
+
+```{code-cell} ipython3
+from typing import List
+from flytekit import task
+
+@task
+def mean(values: List[float]) -> float:
+    return sum(values) / len(values)
+```
+
+As you can see, a task is just a regular Python function that's decorated
+with {py:func}`@task <flytekit.task>`. We can run this function just like any
+other Python function:
+
+```{code-cell} ipython3
+mean(values=[float(i) for i in range(1, 11)])
+```
+
+```{important}
+There are two important things to note here:
+
+- Most of the Flyte tasks you'll ever write can be executed locally.
+- Tasks must be invoked with keyword arguments.
+```
+
```

Comment:
I was expecting to see some "demonstration" about the initial sentence, eg. "this is how a running task looks like in the Kubernetes backend. kubectl get pod..." to also have something for platform engineers looking to correlate Flyte concepts to actual infrastructure (containers in this case) resources

Reply:
can you say more on what content you expect here? What is a "demonstration" in this context?

for some background, we're optimizing this getting started guide for task/workflow authors (i.e. not platform engineers)

IMO it's way too early in this journey to start using `kubectl` commands

Reply:
Thanks for the clarification on the audience. In that case, it's fine as it is